ci/utils.sh

```diff
@@ -54,3 +54,13 @@ architecture() {
             ;;
     esac
 }
+
+# 1 - false
+# 0 - true
```

Comment:
I think you can just remove this comment. As @okdana said, `1` for `false` is standard in shell scripts.